conf/modules.config

```diff
@@ -96,6 +96,45 @@ process {
         ].join(' ').trim()
     }
 
+    withName: SEQKIT_SEQ_FILTER {
+        ext.prefix = { "${meta.id}_cleaned.faa" }
+        publishDir = [
+            path: { "${params.outdir}/function/interproscan/" },
+            mode: params.publish_dir_mode,
+            enabled: { params.run_function_interproscan },
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
+        ext.args   = [
+            "--gap-letters '* \t.' --remove-gaps"
+        ].join(' ').trim()
+    }
+
+    withName: INTERPROSCAN_DATABASE {
+        publishDir = [
+            path: { "${params.outdir}/databases/interproscan/" },
+            mode: params.publish_dir_mode,
+            enabled: params.save_db,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
+    }
+
+    withName: INTERPROSCAN {
+        ext.prefix = { "${meta.id}_interproscan.faa" }
```

Comment:
Is this meant to have the file suffix at the end?

Reply:
Yes, other wise we will have conflicting names in the work dir with suffix `*.interproscan.tsv` and because ampcombi recognises atm files with '*.faa.tsv'